tests/torch/test_compile_torch.py

```diff
@@ -1144,7 +1147,12 @@ def test_shape_operations_net(
     check_float_array_equal,
 ):
     """Test a pattern of reshaping, concatenation, chunk extraction."""
-    net = ShapeOperationsNet(is_qat)
+    net = model(is_qat)
```

Comment:
small detail but in most of our tests we use `model_class` for the parameter and `model` for the actual instance (instead of `net` here), would be great if you could update this test with that !

Reply:
Yes! Let's keep it consistent. I will update.

Reply:
Done